cargo-bpf/src/build.rs

```diff
@@ -71,7 +71,13 @@ impl From<Error> for CommandError {
     }
 }
 
-fn build_probe(cargo: &Path, package: &Path, target_dir: &Path, probe: &str) -> Result<(), Error> {
+fn build_probe(
```

Comment:
I'd prefer no `Option` parameters to functions, and following the `build_probe_with_kernel` pattern for direct invocation, along with the wrapper `build_probe` that preserves the existing logic, same as above.